.github/workflows/_build_base.yaml

```diff
@@ -67,6 +67,16 @@ jobs:
           driver-opts: |
             image=moby/buildkit:v0.12.1
 
+      - name: Print BASE_IMAGE
```

Comment:
No need to specifically print it out, as it will be part of the log of the next step.